src/views/system/file/main/FileMain/index.vue

```
@@ -271,6 +272,24 @@ const handleSelectFile = (item: FileItem) => {
+// 全选/取消全选
+const handleSelectAll = (checked: boolean) => {
+  if (checked) {
+    // 全选：将当前页所有未选中的文件添加到选中列表
+    fileList.value.forEach((item) => {
+      if (!selectedFileIds.value.includes(item.id)) {
+        selectedFileList.value.push(item)
+      }
+    })
+  } else {
+    // 取消全选：移除当前页所有已选中的文件
+    const currentPageIds = new Set(fileList.value.map(item => item.id))
```

ESLint

[error] Expected parentheses around arrow function argument.

(style/arrow-parens)